sdk/storage/storage-blob/src/PageBlobClient.ts

```diff
@@ -649,8 +649,8 @@ export class PageBlobClient extends BlobClient {
     count: number,
     options: PageBlobUploadPagesFromURLOptions = {}
   ): Promise<Models.PageBlobUploadPagesFromURLResponse> {
-    options.accessConditions = options.accessConditions || {};
-    options.sourceModifiedAccessConditions = options.sourceModifiedAccessConditions || {};
+    options.conditions = options.conditions || {};
```

Comment:
Is this align to other languages, for me it looks like conditions -> destConditions || destinationConditions would be better.

Reply:
Yes please see the forwarded meeting notes on Storage ETag support. This is mostly about changing the `accessCondition` to not be nested and with name `conditions` which is more about conditional requests not about destination.

While renaming that, I found `sourceModifiedAccessConditions` a bit too long so I renamed it too. We could revert the renaming to `sourceConditions` if it caused confusions.

Reply:
Just checked current .NET code.  They are not consistent either: only one place uses `destinationAccessCondition` while other places uses `accessCondition`. @tg-msft are you going to make some update based on decision from the meeting last week?

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/8275 switches to `conditions` as the default name and occasionally `sourceConditions`/`destinationConditions` for some copy operations.

Reply:
Thanks Ted! For JS I'd prefer `conditions` everywhere and additional `sourceCondtions` for copy from url operations